src/scss/abstracts/_checkbox.scss

```diff
@@ -0,0 +1,110 @@
+/*EXAMPLE
+<div id="gtb">
+  <div class="growth-tools-brand__sections--checkbox">
+    <h3>CheckBox</h3>
+    <div class="gtb-checkbox-wrapper">
+      <label class="gtb-checkbox-label">
+        <input class="gtb-checkbox-input" type="checkbox" name="one" value="one" />
+        <span class="gtb-checkbox__box">
+          <img src="/assets/images/icons/check.svg" alt="check icon" class="gtb-checkbox__checkmark" />
+        </span>
+        Checkbox Label
+      </label>
+    </div>
+    <div class="gtb-checkbox-wrapper">
+      <label class="gtb-checkbox-label">
+        <input class="gtb-checkbox-input" type="checkbox" name="two" value="two" />
+        <span class="gtb-checkbox__box">
+          <img src="/assets/images/icons/check.svg" alt="check icon" class="gtb-checkbox__checkmark" />
+        </span>
+        Checkbox Label
+      </label>
+    </div>
+  </div>
+</div>
+
+
+*/
+#gtb {
```

Comment:
I'm not sure about using a CSS ID for this selector. IDs should only appear once on a page. What were your thoughts on adding the ID wrapper?